packages/superset-ui-chart/src/models/ChartProps.ts

```diff
@@ -37,7 +35,7 @@ export interface ChartPropsConfig {
    * Formerly called "filters", which was misleading because it is actually
    * initial values of the filter_box and table vis
    */
-  initialValues?: InitialValues;
+  activeFilters?: DataRecordFilters;
```

Comment:
Renamed `initialValues` to `activeFilters` as it's not super clear what things' initial values it is. Superset frontend will update accordingly.

Reply:
Reverted it back just to be safe.